frontend/src/components/layout/navigation/MobileNavigation.module.scss

```diff
@@ -3,7 +3,9 @@
 
 .mobile-menu {
   width: 100%;
-  position: relative;
+  position: sticky;
+  top: 76px; // threshold for mobile menu - height of header
```

Comment:
One worry I have with this is that this "happens" to be the width of the header, but we're never explicitly setting it to this value. In other words, if there's anything that might affect the height of the header, the menu will suddenly be out of place.

[Reading up on sticky positioning](https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky) a bit, I'm wondering if a different solution might be possible - one that potentially also fixes the issue with the `height: 100%` on Layout's `.wrapper`. Specifically:

> Note that a sticky element "sticks" to its nearest ancestor that has a "scrolling mechanism" (created when overflow is hidden, scroll, auto, or overlay), even if that ancestor isn't the nearest actually scrolling ancestor.

So, given that we don't want the menu to be positioned below the header, but also to overlap the content, I'm wondering if it might work if in `<Layout>`, we wrap `<MobileNavigation>`, `.content` and `footer.footer` (or possibly not even the latter) in a div with `height: 100%; overflow: auto`? As I understand it, that div will then always be below the header (so without needing `top` with a magic number), while `<MobileNavigation>` will be sticky to the top of that div? Not sure if it will work, but might be worth a try.

Reply:
Taking another look to see if we can solve this issue - thanks!